### DISCUSSION TITLE
Isn't the selection colour too weak?

### DISCUSSION THREAD

**Discussion by HubKing** [NvChad/NvChad#2622] [Q&A] [answered]
I have tested a lot of the in-built themes but they all seem to have very weak contrast for the selection colour. For example, see below. I don't know about you, but I can barely notice which line is selected. 

If I am not the only person who has this problem, what's the solution? Should I manually edit themes? Is there any easier way to make selection more noticeable?

![image](https://github.com/NvChad/NvChad/assets/10900489/689959d3-3a82-436f-9c1f-31eb04be4dbb)

**Comment by siduck** [comment, marked as answer]
https://nvchad.com/docs/config/theming
